MapboxVision/src/main/java/com/mapbox/vision/video/videosource/camera/SurfaceVideoRecorder.kt

```diff
@@ -15,6 +15,25 @@ internal interface VideoRecorder {
     fun startRecording(path: String)
     fun stopRecording()
     fun release()
+
+    class EmptyImpl : VideoRecorder {
```

Comment:
we already have `DummyImageSaver` and `DummyEventManager`. Lets keep the same style. `DummyVideoRecorder`

Reply:
done